docs/server/settings.rst

```diff
@@ -281,33 +281,59 @@ Translation environment configuration settings.
 
 .. setting:: POOTLE_TM_SERVER
 
-.. versionadded:: 2.7
-
 ``POOTLE_TM_SERVER``
+  .. versionadded:: 2.7
+
+  .. versionchanged:: 2.7.3
+
+     Added the ``WEIGHT`` option. Also added another default TM used to import
+     external translations from files.
+
+
   Default:
 
   .. code-block:: python
 
     {
-        'default': {
+        'local': {
             'ENGINE': 'pootle.core.search.backends.ElasticSearchBackend',
             'HOST': 'localhost',
             'PORT': 9200,
             'INDEX_NAME': 'translations',
+            'WEIGHT': 1,
+            'MIN_SCORE': 'AUTO',
+        },
+        'external': {
+            'ENGINE': 'pootle.core.search.backends.ElasticSearchBackend',
+            'HOST': 'localhost',
+            'PORT': 9200,
+            'INDEX_NAME': 'external-translations',
+            'WEIGHT': 0.9,
             'MIN_SCORE': 'AUTO',
         },
     }
 
+
   This is configured to access a standard Elasticsearch setup.  Change the
   settings for any non-standard setup.  Change ``HOST`` and ``PORT`` settings
   as required.
 
   Use ``MIN_SCORE`` to set the Levenshtein Distance score.  Set it to ``AUTO``
-  so that Eslasticsearch will adjust the required score depending on the length
+  so that Elasticsearch will adjust the required score depending on the length
   of the string being translated. Elasticsearch documentation provides further
   details on `Fuzzy matching
   <https://www.elastic.co/guide/en/elasticsearch/reference/current/common-options.html#fuzziness>`_.
 
+  The default ``local`` TM is automatically updated every time a new
+  translation is sent. The other TMs are not automatically updated so they can
+  be trusted to provide selected high quality translations.
+
+  ``INDEX_NAME`` every TM server must have its own unique index.
+  
+  ``WEIGHT`` provides a weighting factor to alter the final score for TM
```

Comment:
Use ``.. setting:: POOTLE_TM_SERVER-WEIGHT`` and reference it with ``:setting:`WEIGHT <POOTLE_TM_SERVER-WEIGHT>```.  Same with the other options and references that have been introduced.